cmd/controller/app/options/options.go

```diff
@@ -99,5 +110,20 @@ func (s *ControllerOptions) AddFlags(fs *pflag.FlagSet) {
 }
 
 func (o *ControllerOptions) Validate() error {
+	switch o.ACMEDNS01CheckMethod {
+	case ACMEDNS01CheckViaDNSLookup:
```

Comment:
👍 this also doesn't quite do the same thing, although it's a no-op. Go doesn't fallthrough by default, so this is actually *not* falling through to the 'break' line below (although that said, nothing else matches so it is meaningless, but I guess it could lead to something bad in future 🙄)